### PR TITLE
BCMOHAD-20479 - Various bug fixes.

### PR DESCRIPTION
**Pull request by dcstuff**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: MAiD - Case Management App/force-app/main/default/objects/Case/recordTypes/Error_Log.recordType-meta.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <fullName>Error_Log</fullName>
     <active>true</active>
     <businessProcess>Error Log Support Process</businessProcess>
-    <label>Error Log</label>
+    <label>MAiD Error Log</label>
```
Comment (cgijeffolsen) [inline MAiD - Case Management App/force-app/main/default/objects/Case/recordTypes/Error_Log.recordType-meta.xml:6]: This was updated to deal with MAiD concern that Error Log was the default, changing this label allowed MAiD Case to return to being the default in our testing. @RanadheerRG

File: MAiD - Case Management App/force-app/main/default/permissionsets/MAiD_Analyst.permissionset-meta.xml [hunk 2]
```diff
@@ -10,8 +10,13 @@
         <readable>true</readable>
```
Comment (cgijeffolsen) [inline MAiD - Case Management App/force-app/main/default/permissionsets/MAiD_Analyst.permissionset-meta.xml:10]: These fields were found to be absent for MAiD teams afer View All permission was removed.

**Review by cgijeffolsen — APPROVED** [review]
Approved using API v59 @dcstuff

**Review by lilianenassar-bib — APPROVED** [review]
Approved

**Pull request merged**